src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -70,7 +72,7 @@
     private static final Logger logger = Logger.getLogger(GhprbTrigger.class.getName());
 
     private static final ExecutorService pool = Executors.newFixedThreadPool(
```

Comment:
Do you have tests, which are baking this change? We are running Jenkins with 32 Cores, which would mean 32 parallel requests to github, all waiting for github's api to return. This just feels like hammering with a very big hammer on a very tiny nail :)